iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/utils/CommonDateTimeUtils.java

```diff
@@ -42,7 +42,8 @@ public static long convertMilliTimeWithPrecision(long milliTime, String timePrec
       default:
         break;
     }
-    return result;
+    // To avoid integer overflow
+    return result < 0 ? Long.MAX_VALUE : result;
```

Comment:
It is better to print a log no lower than the warning level. It is even better to also print the stack traces.
Otherwise, any problem will be unable to be traced.

Reply:
Resolved. A warning log has been added when integer overflow.